controller.go

```diff
@@ -259,11 +276,25 @@ func (c *controller) post(path, op string, params url.Values) (interface{}, erro
 	return parsed, nil
 }
 
+func (c *controller) getQuery(path string, params url.Values) (interface{}, error) {
+	return c._get(path, params)
+}
+
 func (c *controller) get(path string) (interface{}, error) {
+	return c._get(path, nil)
+}
+
+func (c *controller) _get(path string, params url.Values) (interface{}, error) {
```

Comment:
Given that `getQuery` and `_get` do the same thing with name signature, why not replace `getQuery` with the existing `_get` and then have `get` call `getQuery`?